Cargo.toml

```
@@ -31,3 +31,8 @@ rand_core = "0.5.1"
 zcash_history = "0.2"
 zcash_primitives = "0.2"
 zcash_proofs = "0.2"
+
```

```suggestion

[profile.dev]
panic = 'abort'

[profile.test]
panic = 'abort'

[profile.bench]
panic = 'abort'

```
However, [`panic = 'abort'` will be ignored](https://doc.rust-lang.org/nightly/cargo/reference/profiles.html#panic) for `profile.test` and `profile.bench` unless the [panic-abort-tests](https://doc.rust-lang.org/nightly/cargo/reference/unstable.html#panic-abort-tests) unstable flag is set, so I suggest also enabling that.

---

We can't enable that flag on stable Rust. (Actually, we can, but it requires cheating and subverting the compiler using a secret incantation.) Since we're always compiling in `--release` this can change can wait I think.

---

(Marking as resolved after discussing in video chat.)